code/game/objects/structures/signs/signs_interactive.dm

```diff
@@ -24,3 +24,50 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/calendar, 32)
 		. += span_info("Events:")
 		for(var/holidayname in SSevents.holidays)
 			. += span_info("[holidayname]")
+
+GLOBAL_LIST_EMPTY(map_delamination_counters)
+
+/obj/structure/sign/delamination_counter
+	name = "delamination counter"
+	desc = "A pair of flip signs describe how long it's been since the last delamination incident."
+	icon_state = "days_since_explosion"
+	var/since_last = 0
+
+MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/delamination_counter, 32)
+
+/obj/structure/sign/delamination_counter/Initialize(mapload)
+	. = ..()
+	if (mapload)
+		GLOB.map_delamination_counters += src
+	else
+		update_count(SSpersistence.rounds_since_engine_exploded)
+
+/obj/structure/sign/delamination_counter/proc/update_count(new_count)
+	since_last = min(new_count, 99)
+	build_overlays()
+
+/obj/structure/sign/delamination_counter/proc/build_overlays()
+	overlays = list()
+
+	var/ones = since_last % 10
+	var/mutable_appearance/ones_overlay = mutable_appearance('icons/obj/signs.dmi', "days_[ones]")
+	ones_overlay.pixel_x = 4
+	overlays += ones_overlay
+
+	var/tens = (since_last / 10) % 10
+	var/mutable_appearance/tens_overlay = mutable_appearance('icons/obj/signs.dmi', "days_[tens]")
+	tens_overlay.pixel_x = -5
+	overlays += tens_overlay
```

Comment:
This breaks emissive blocker overlays
Why not just use update_overlays/update_appearance?